src/index.js

```diff
@@ -53,4 +54,7 @@ if (config.plugAndPlay) {
     handleUnknownDeviceData();
 }
 
+const statisticSavers = {};
```

Comment:
it is just used for better testability of component for now.
but maybe it will be useful for reading data in future